doc/userguide/configuration/suricata-yaml.rst

```diff
@@ -2912,7 +2912,7 @@ The Teredo decoder can be disabled. It is enabled by default.
         ports: $TEREDO_PORTS # syntax: '[3544, 1234]'
 
 Using this default configuration, Teredo detection will run on UDP port
-3544. If the `ports` parameter is missing, or set to `any`, all ports will be
+1.    If the `ports` parameter is missing, or set to `any`, all ports will be
```

Comment:
Was this change supposed to be part of this work? Couldn't find any other changes related to Teredo, nor commits highlighting relevant changes.

Reply:
nah, thanks. I must have misadded this one. Thanks.